src/like-service/database/migrations/2023_07_13_073935_create_like_table.php

```diff
@@ -13,10 +13,10 @@ class CreatePasswordResetsTable extends Migration
      */
     public function up()
     {
-        Schema::create('password_resets', function (Blueprint $table) {
-            $table->string('email')->index();
-            $table->string('token');
-            $table->timestamp('created_at')->nullable();
+        Schema::create('like', function (Blueprint $table) {
+            $table->integer('postId', false);
+            $table->integer('UserId', false);
```

Comment:
[NIT] postId's first letter is lower case vs UserId's letter is upper case. Please unify them

Reply:
Changed to lower case